index.js

```diff
@@ -4,6 +4,46 @@ const moment = require("moment")
 const bugsnag = require('bugsnag')
 const Discord = require('discord.js')
 const Table = require("ascii-table")
+const cron = require('cron');
+
+// Live Updates Cron Job
+let liveStatChannelIds = [];
+let liveMatchStats = [];
+var cronJob = cron.job("0 */1 * * * *", async function () {
+  try {
+    console.info('Poll for goll..');
+    const response = await got('http://worldcup.sfg.io/matches/current', {
+      json: true
+    })
+    let message = ""
+    const responseMatches = response.body
+    responseMatches.forEach(newMatchStats => {
+      let matchId = newMatchStats.home_team.code + newMatchStats.away_team.code
+      let oldMatchStats = liveMatchStats.find(liveMatch => { return liveMatch.id == matchId })
+      if (oldMatchStats) {
+        // Check if score has changed to update feed
+        if (oldMatchStats.homeGoals != newMatchStats.home_team.goals || oldMatchStats.awayGoals == newMatchStats.away_team.goals) {
```

Comment:
should this not be `oldMatchStats.homeGoals != newMatchStats.home_team.goals || oldMatchStats.awayGoals != newMatchStats.away_team.goals`?

Also maybe its better to do `oldMatchStats.homeGoals > newMatchStats.home_team.goals || oldMatchStats.awayGoals > newMatchStats.away_team.goals`